### PR TITLE
Improve rebalance and add pause/resume exporters in camunda.mk

### PR DESCRIPTION
**Pull request by chDame**
Add to make target to pause/resume exporters that will be useful for dual region failOver. And make the rebalance a oneline without sleep or kill. Cleaner

### CODE REVIEW

**Review by upgradingdave — APPROVED** [review]

File: include/camunda.mk [hunk 1]
```diff
@@ -59,16 +59,22 @@ update:
...
 curl-rebalance:
 	curl -X POST http://localhost:9600/actuator/rebalance
 
+.PHONY: pause-exporters
+pause-exporters:
+	kubectl exec $$(kubectl get pod --namespace $(namespace) --selector="app=camunda-platform,app.kubernetes.io/component=zeebe-gateway,app.kubernetes.io/instance=camunda,app.kubernetes.io/managed-by=Helm,app.kubernetes.io/name=zeebe-gateway,app.kubernetes.io/part-of=camunda-platform" --output jsonpath='{.items[0].metadata.name}') --namespace $(namespace) -c zeebe-gateway -- curl -i localhost:9600/actuator/exporting/pause -XPOST
+	
+.PHONY: resume-exporters
+resume-exporters:
+	kubectl exec $$(kubectl get pod --namespace $(namespace) --selector="app=camunda-platform,app.kubernetes.io/component=zeebe-gateway,app.kubernetes.io/instance=camunda,app.kubernetes.io/managed-by=Helm,app.kubernetes.io/name=zeebe-gateway,app.kubernetes.io/part-of=camunda-platform" --output jsonpath='{.items[0].metadata.name}') --namespace $(namespace) -c zeebe-gateway -- curl -i localhost:9600/actuator/exporting/pause -XPOST
```
Comment (upgradingdave) [inline include/camunda.mk]: Hi @chDame I caught and fixed a small bug here. I renamed `pause` to `resume`

**Review by upgradingdave — APPROVED** [review]

**Comment by upgradingdave** [comment]
I tested and was able to rebalance brokers and pause/resume exporters, looks good 😎

**Comment by falko** [comment]
@chDame @upgradingdave This change doesn't work for me:
```
$ make rebalance-leaders 
kubectl exec $(kubectl get pod --namespace camunda --selector="app=camunda-platform,app.kubernetes.io/component=zeebe-gateway,app.kubernetes.io/instance=camunda,app.kubernetes.io/managed-by=Helm,app.kubernetes.io/name=zeebe-gateway,app.kubernetes.io/part-of=camunda-platform" --output jsonpath='{.items[0].metadata.name}') --namespace camunda -c zeebe-gateway -- curl -i localhost:9600/actuator/rebalance -XPOST
error: Internal error occurred: error executing command in container: failed to exec in container: failed to start exec "ae2dd0a850568d097a5157e487fcf4d2125a69e165cefbacc8c43607bb72e8d3": OCI runtime exec failed: exec failed: unable to start container process: exec: "curl": executable file not found in $PATH: unknown
make: *** [../../../../camunda-8-helm-profiles/include/camunda.mk:63: rebalance-leaders] Error 1
```

**Comment by upgradingdave** [comment]
Hi @falko , I have a working GCP cluster and just tested and seems to work fine for me. 

The path to `include/camunda.mk` is calculated using a `$(root)` makefile variable defined in the Makefile [here](https://github.com/camunda-community-hub/camunda-8-helm-profiles/blob/88428a80c2d07ec7f5104858b24f5d6904f47362/google/ingress/nginx/tls/Makefile#L35)

The error message you pasted shows this as the path: `../../../../camunda-8-helm-profiles/include/camunda.mk:63`

But I think the path should be this: `../../../../include/camunda.mk:63`

So it seems in your environment that there's an extra `camunda-8-helm-profiles/` directory in the `$(root)` path? Can you double check that you don't have any local changes that might be causing confusion?

**Comment by falko** [comment]
@upgradingdave it has nothing to do with the local files. `curl` doesn't seem to be available in the container: `"curl": executable file not found in $PATH`. I just also got the same problem with `zbctl`. Maybe the latest Zeebe images have been cleaned up to reduce attack surface?

**Comment by falko** [comment]
Well, since there is an official curl image, I'd see that as the natural choice. Nixery would be a solution for `zbctl`: https://search.nixos.org/packages?channel=23.05&show=zbctl&from=0&size=50&sort=relevance&type=packages&query=zbctl

**Comment by falko** [comment]
#46 already has the link to a rebalancing cronjob example

**Pull request merged**